### ISSUE TITLE
[Feature Request] Column dividers in data tables

### ISSUE THREAD

**Issue by iec989** [vuetifyjs/vuetify#3364]
### New Functionality
To clearly separate columns with a line between them

### Improvements
Instead of adding custom CSS for the table, adding a HTML attribute such as "grid-lines" , should paint the dividers between columns

### Bugs or Edge Cases it Helps Avoid
Any bugs coming from building custom CSS on the standard vuetify framework, that may stop working without one mentioning it by a Vuetify update.

**Comment by chewy94** [comment]
Not sure this is material spec, but also I do not see why this cannot be implemented

**Comment by nicoepp** [comment]
I don't now if they are in the spec but at least one can observe them in the example pictures of the material design docs.

https://material.io/design/components/data-tables.html#behavior

![](https://storage.googleapis.com/spec-host-backup/mio-design%2Fassets%2F1mhrI0DNGU3LRbZtzJ1brL-AE99dTVi9i%2Fcomponents-datatables-interaction1.png)

**Comment by borisd13** [comment]
In the meantime, if anyone needs to add it, you can just use the following.

```
<style>
table th + th { border-left:1px solid #dddddd; }
table td + td { border-left:1px solid #dddddd; }
</style>
```

_**Note**: Solution adapted from Erasmus https://stackoverflow.com/questions/3313456/css-borders-between-table-columns-only/3313493_

**Comment by blalan05** [comment]
Available in the 2.0 alpha.

**Comment by dblinov** [comment]
> Available in the 2.0 alpha.

@blalan05 Where can I find example of style setting to get divider?

**Comment by naffarn** [comment]
> Available in the 2.0 alpha.

I can't find anything in the documentation or release notes about being able to use column separators in a table, was this actually implemented?

**Comment by renishbabu876** [comment]
I also need this feature
